pingdom-checks/provider-directory.ping

```diff
@@ -0,0 +1,18 @@
+# -*- sh -*-
+
+SANDBOX_PROVIDER_DIRECTORY_API_KEY=$(get-secret "/dvp/lab/provider-directory/api-key")
+PROVIDER_DIRECTORY_SLACK_ID="${SLACK_VASDVP_MONITORING},${SLACK_API_ALARMS_DATA_QUERY}"
```

Comment:
So alarms for Provider Directory will go to the DQ alarms channel as well? cc @bschofield-va